File: Programmers/Level1/완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer ="";
        HashMap<String, Integer> map = new HashMap<String, Integer>();
        for(String i : participant){
            if(map.containsKey(i)){
            map.put(i, map.get(i)+1);
        }else{
            map.put(i, 1);
            }
        }

        for(String i : completion){
            map.put(i, map.get(i)-1);
        }

        for(String i : participant){
            if(map.get(i)!=0){
                answer=i;
                break;
            }
        }
            return answer;
    }
}

File: Programmers/Level1/완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):

    dic = {}
    
    for i in range(0, len(participant)):
        if dic.get(participant[i]) != None:
            dic[participant[i]] = dic.get(participant[i])+1
        else:
            dic[participant[i]] = 1
    
    for i in range(0, len(completion)):
        dic[completion[i]] = dic.get(completion[i])-1
            
    for i in dic:
        if dic.get(i) !=0:
            answer = i

    return answer
